Recrutify/Recrutify.Host/Startup.cs

```diff
@@ -75,6 +76,13 @@ public void ConfigureServices(IServiceCollection services)
 
             services.Configure<RouteOptions>(options => options.LowercaseUrls = true);
 
+            services.AddAuthorization(options =>
```

Comment:
Put it after line where AddIdentityServerAuthentication, to keep all logic roles together